file-formats/intf/zif_aff_types_v1.intf.abap

```diff
@@ -0,0 +1,34 @@
+INTERFACE zif_aff_types_v1 PUBLIC .
+
+  types:
+    begin of enum abap_language_version ,
+      standard,
+      cloud_development,
+    end of enum abap_language_version.
```

Comment:
Should we specify the ABAP values that are used in the back end? Otherwise it will be hard to find the mapping between enum and ABAP value.
```suggestion
  types:
    begin of enum abap_language_version ,
      standard value is intial,
      cloud_development value '5',
    end of enum abap_language_version.
```

Reply:
Make sense to me